splunktaucclib/config.py

```diff
@@ -365,7 +356,8 @@ def load_value(self, endpoint_id, item_name, fname, fval):
 
     def try_fix_corrupted_json(self, corrupted_json, value_err):
         """
-        When L3 (one of our customers) was testing, they encountered a bug that 'access_token_encrypted' or 'refresh_token' got corrupted when it was saved in the conf file
+        When L3 (one of our customers) was testing, they encountered a bug that 'access_token_encrypted'
```

Comment:
Can you please remove that mention of our customer?

Reply:
Updated the docstring